torch/_functorch/aot_autograd.py

```diff
@@ -672,7 +672,7 @@ def to_fun(t):
         if isinstance(t, Tensor):
             if t in memo:
                 return memo[t]
-            r = torch._to_functional_tensor(t, mirror_autograd_meta=True)
+            r = torch._to_functional_tensor(t, mirror_autograd_meta=True) if not t.is_nested else t
```

Comment:
let me know if/when you want some help with this - I think in theory it shouln't be too hard to get functionalization working (there are probably some view ops specific to nested tensor, that we need to define "inverses" for)

Reply:
thanks! for now, I'm avoiding it to get benchmark results but I will definitely need your help at some point :)